site/content/blog/getting-started-with-ytt.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Getting Started With ytt, Part 1"
+slug: getting-started-with-ytt
+date: 2022-10-12
+author: Varsha Munishwar
+excerpt: "Are you new to ytt and wondering where to start?
+Here is an easy step-by-step tutorial that can introduce you to ytt and help you get started quickly.
+"
+image: /img/ytt.svg
+tags: ['ytt', 'getting started', 'tutorials']
+---
+
+#### Welcome to the series of ytt getting started tutorials!
+
+The part 1 of this series introduces you to [ytt](https://carvel.dev/ytt/) and helps you get started quickly.
```

Comment:
nit: _I think_ you could start without "The" so that it reads more smoothly.
Consider:
> Part 1 of this series...